src/ios/CDVInAppBrowser.h

```diff
@@ -63,24 +63,31 @@
 @property (nonatomic, assign) BOOL suppressesincrementalrendering;
 @property (nonatomic, assign) BOOL hidden;
 @property (nonatomic, assign) BOOL disallowoverscroll;
+@property (nonatomic, assign) BOOL validatessl;
 
 + (CDVInAppBrowserOptions*)parseOptions:(NSString*)options;
 
 @end
 
-@interface CDVInAppBrowserViewController : UIViewController <UIWebViewDelegate, CDVScreenOrientationDelegate>{
+@interface CDVInAppBrowserViewController : UIViewController <UIWebViewDelegate, CDVScreenOrientationDelegate, NSURLConnectionDataDelegate>{
```

Comment:
The approach in the iOS community seems to be to let lines run as long as needed and let Xcode do the formatting. I know with some of our other forks that we tend to keep to the style of the original code. Not really hard over either way though, and I could definitely see line-wrapping it to make it look nicer on Github.